app/2.3.x/external-plugins.md

```diff
@@ -0,0 +1,248 @@
+---
+title: External Plugins Support
+---
+
+## Introduction
+
+External plugins are those that run on a process separate from {{site.base_gateway}} itself,
```

Comment:
I would begin this by saying "Kong supports Lua plugins natively. Plugins written in other languages need to be served by a separate process, that we call _plugin server_."

Reply:
done